compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -4371,6 +4372,87 @@
                     "type": "ExpressionNode"
                 }
             ]
+        },
+        {
+            "name": "ResourceMethodCallActionNode",
+            "base": "ActionNode",
+            "kind": "RESOURCE_METHOD_CALL_ACTION",
+            "attributes": [
+                {
+                    "name": "expression",
+                    "type": "ExpressionNode"
+                },
+                {
+                    "name": "rightArrowToken",
+                    "type": "Token"
+                },
+                {
+                    "name": "slashToken",
+                    "type": "Token"
+                },
+                {
+                    "name": "resourceAccessPath",
+                    "type": "Node",
+                    "occurrences": "MULTIPLE"
```

Comment:
```suggestion
                    "occurrences": "MULTIPLE_SEPARATED"
```

Reply:
If we use `MULTIPLE_SEPARATED` the formatter adds spaces after the separator. We have also used `"occurrences": "MULTIPLE"` for relativeResourcePath in `FunctionDefinitionNode`.